docs/classes.list.js

```diff
@@ -43,36 +43,55 @@ export const backgroundColor = [
   'bg-inherit',
   'bg-current',
   'bg-transparent',
```

Comment:
These are not in `semantic.yml` file, not sure if I should keep them or not. Not mentioned in the spreadsheet either.

Reply:
Hm. They are part of [color rules in warp-ds/drive](https://github.com/warp-ds/drive/blob/alpha/src/_rules/color.js), so I don't think we should remove them. Wdyt @Skadefryd?